src/test/java/com/ksm/wstbackoffice/controller/AddressControllerIT.java

```diff
@@ -19,16 +23,24 @@
         @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/addressDelete.sql")
 })
 public class AddressControllerIT extends BaseControllerIT {
-    @Test
-    public void findByIdTest() {
+    public static Stream<Arguments> findByIdTestArguments() {
+        return Stream.of(
+                Arguments.of(1L, 200),
+                Arguments.of(-1L, 400),
+                Arguments.of(0L, 400),
+                Arguments.of(999999999999999999L, 404)
```

Comment:
Instead of hard coding some big value you can pass already existing constant Long.MAX_VALUE, if I remember well

Reply:
done